### ISSUE TITLE
links embedded in metadata should be underlined

### ISSUE THREAD

**Issue by jcoyne** [sul-dlss/purl#1024]
https://my2.siteimprove.com/Accessibility/1064058/NextGen/Issue/1?ruleId=62&pageSegments=&issueKind=1&exceptTags=1,2&conformance=&siteTargetIssueKinds=1,2&wcagVersion=22&siteTarget.wcagVersion=22&siteTarget.conformanceLevels=0,1,3,4&siteTarget.issueKinds=1,2

In "Use and reproduction" on https://purl.stanford.edu/bb537hc4022
